front/src/index.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './pages/App/App';
 import { BrowserRouter } from 'react-router-dom';
+import './styles/main.scss';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const root = ReactDOM.createRoot(document.getElementById('root') as HTMLDivElement);
```

Comment:
I think `as HTMLElement` will be good enough: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-dom/client.d.ts#L34